packages/story-editor/src/components/panels/design/captions/captions.js

```diff
@@ -127,11 +126,11 @@ function CaptionsPanel({ selectedElements, pushUpdate }) {
   );
 
   const handleChangeTrack = useCallback(
-    (attachment) => {
+    ({ src = '', id }) => {
       const newTracks = {
-        track: attachment?.url,
-        trackId: attachment?.id,
-        trackName: attachment?.filename,
+        track: src,
+        trackId: id,
+        trackName: src.split('/').pop(),
```

Comment:
Could use use [getFileNameFromUrl](https://github.com/google/web-stories-wp/blob/main/packages/media/src/getFileNameFromUrl.js) here?

Reply:
The existing code `attachment?.filename` gives the file name with extension ( for example caption.vtt ) however, [getFileNameFromUrl](https://github.com/google/web-stories-wp/blob/main/packages/media/src/getFileNameFromUrl.js) will return only the file name without extension.

Reply:
Don‘t think we need to store trackName here in the first place.